lib/pure/typetraits.nim

```diff
@@ -57,6 +57,16 @@ proc supportsCopyMem*(t: typedesc): bool {.magic: "TypeTrait".}
   ## This trait returns true iff the type ``t`` is safe to use for
   ## `copyMem`:idx:. Other languages name a type like these `blob`:idx:.
 
+proc `==`*(t1, t2: typedesc): bool =
+  ## Returns wheter ``t1``, ``t2`` are the same type; this is different
+  ## from ``t1 is t2`` since the latter supports concepts & inheritance.
+  runnableExamples:
+    type T=int
```

Comment:
Style nitpick: space before and after ``=``.

Reply:
done